model-optimizer/mo/utils/cli_parser.py

```diff
@@ -118,7 +118,7 @@ def readable_file_or_dir(path: str):
     :return: path if the file/directory is readable
     """
     if not os.path.isfile(path) and not os.path.isdir(path):
-        raise Error('The "{}" is not existing file or directory'.format(path))
+        raise Error('The "{}" file or directory does not exist'.format(path))
```

Comment:
please do not change the error message. Due to this change mo_params_tests.mo_args_test.NegativeInputs.test_nonexisiting_input_modelis failing

Reply:
Removed.